scripts/convert-docs.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { writeFile } = require('fs/promises');
+const apiDocs = require('../src/api-docs');
+
+(async () => {
+  // Remove redundant `default` property
+  const docsObj = apiDocs.default || apiDocs;
```

Comment:
This may be problematic if we create a default key on the root of api-docs.
It shouldn't be a part of the object since openapi does not have a root default key but we would be ignoring this error.

Reply:
Actually this is a way of retrieving the main object from the `api-docs.js` file, that is exported like this:

https://github.com/HathorNetwork/hathor-wallet-headless/blob/a793c26ed03b895650089b3f4761a150bed454ba/src/api-docs.js#L3822

The `|| apiDocs` was just a way of protecting this script against failures, but indeed it would muffle a case where this obejct export syntax was changed and cause confusion on the developer. This was fixed on [18fdf61](https://github.com/HathorNetwork/hathor-wallet-headless/pull/327/commits/18fdf61b1ed26b601641009d83fbc73c45e773c4).

Reply:
This feature was re-written with a different approach, this thread is now obsolete.